### PR TITLE
Move files to plos template.

### PR DESCRIPTION
**Pull request by drvinceknight**

### CODE REVIEW

**Comment by drvinceknight** [comment]
Hey @marcharper and @Nikoleta-v3 if you could take a moment to check that I haven't made a mess of anything that would be great.

PLOS's style file essentially needs everything to be "hard coded" so errors are not impossible...

File: main.tex [hunk 1]
```diff
@@ -1,846 +1,1374 @@
...
+reported in~\cite{Stewart2012} the extortionate strategy Extort-2 gains a large number of
+wins. Notice that the mean \(\SSe\) for Extort-2 is approximately zero, while for
+the always cooperating strategy Cooperator the \(\SSe\) is far from zero. It is
+also clear that ZD-GTFT2 defined as a ZD strategy does not act
+extortionately. This is evident by the fact that it does not rank highly according
+to wins which is due to its value of \(\chi\) being less than 1.
+ZD-GTFT2 is referred to as a ``generous'' ZD strategy, other examples of this
+include ZDGen2 and ZDSet2 defined in~\cite{sep-prisoner-dilemma}. The
+general performance of these will be discussed in
+Section~\ref{sec:numerical-experiments}.
```
Comment (Nikoleta-v3) [inline main.tex]: This doesn't work any more since the sections are not numbered

Comment (Nikoleta-v3) [reply main.tex]: In Section Results?

**Comment by Nikoleta-v3** [comment]
Thanks for your work! I only picked up one issue with referencing one of the sections. Everything else looks fine!

One question; the pdf includes line numbers. Do they ask us to submit it with the line numbers?

File: main.tex (hunk 1 above)
Comment (drvinceknight) [reply main.tex]: Yup, thought I'd caught all of these. Thanks!

**Comment by drvinceknight** [comment]
> Thanks for your work! I only picked up one issue with referencing one of the sections. Everything else looks fine!

Thanks Nik :)

> 
> One question; the pdf includes line numbers. Do they ask us to submit it with the line numbers?

I believe so! I'll double check and turn them of if not :)

**Comment by marcharper** [comment]
lgtm

**Comment by drvinceknight** [comment]
Thanks both, I'll submit this first thing tomorrow :+1: (Famous last words)

**Pull request merged**